pkg/v7patch1/configmap/mock_test.go

```diff
@@ -39,7 +39,11 @@ func (h *helmMock) DeleteRelease(releaseName string, options ...helm.DeleteOptio
 	return nil
 }
 
-func (h *helmMock) EnsureTillerInstalled() error {
+func (h *helmMock) EnsureTillerInstalled(ctx context.Context) error {
```

Comment:
Don't understand how this even worked before.

Reply:
Me neither! :)